fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/jdbc/ConnectionStub.java

```diff
@@ -44,14 +44,12 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
 
     @Override
     public Statement createStatement() throws SQLException {
-        // Stub Only
-        return null;
+        return new StatementStub(this);
```

Comment:
it's weird that my local looks correct - the same as you change. 
and also it's a big surprise to find that the line numbers are totally different from my local...

Reply:
same -  auto generated comments, I added the statement stub back in between a prior commit and this one.  There were 100s of lines of auto-generated comments